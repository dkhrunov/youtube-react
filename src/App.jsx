import React from 'react';
import Home from './containers/Home/Home';
import Watch from './containers/Watch/Watch';
import AppLayout from './components/AppLayout/AppLayout';
import { Switch, Route } from 'react-router-dom';
import './styles/main.scss';

function App() {
	return (
		<AppLayout>
			<Switch>
				<Route path='/' exact component={Home} />
				<Route path='/watch' component={Watch} />
			</Switch>
		</AppLayout>
	);
}

export default App;
