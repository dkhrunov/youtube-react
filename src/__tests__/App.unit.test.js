import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Renders App component', () => {
	test('renders', () => {
		const wrapper = shallow(<App />);
		expect(wrapper).toMatchSnapshot();
	});
});
