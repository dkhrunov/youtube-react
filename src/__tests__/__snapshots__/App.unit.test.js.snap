// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App component renders 1`] = `
<AppLayout>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/watch"
    />
  </Switch>
</AppLayout>
`;
